.eslintrc

```diff
@@ -32,5 +32,13 @@
         "no-var": "error",
         "functional/no-class": "error",
         "functional/prefer-type-literal": "error"
-    }
+    },
+    "overrides": [
```

Comment:
i think this can be added to the non override section, the wildcard is a little too general for an override